sharding/nodesCoordinator/indexHashedNodesCoordinator.go

```diff
@@ -1288,10 +1292,11 @@ func (ihnc *indexHashedNodesCoordinator) getShardValidatorInfoData(txHash []byte
 	return shardValidatorInfo, nil
 }
 
-func (ihnc *indexHashedNodesCoordinator) updateEpochFlags(epoch uint32) {
-	ihnc.flagStakingV4Started.SetValue(epoch >= ihnc.enableEpochsHandler.StakingV4Step1EnableEpoch())
-	log.Debug("indexHashedNodesCoordinator: flagStakingV4Started", "enabled", ihnc.flagStakingV4Started.IsSet())
-
-	ihnc.flagStakingV4Step2.SetValue(epoch >= ihnc.stakingV4Step2EnableEpoch)
-	log.Debug("indexHashedNodesCoordinator: flagStakingV4Step2", "enabled", ihnc.flagStakingV4Step2.IsSet())
+func (ihnc *indexHashedNodesCoordinator) updateEnableEpochsHandler(epoch uint32) error {
```

Comment:
Same comment as the one from shuffler. Pass the expected interface here, no cast and no return error.

Reply:
updated to different interface